test/SourceKit/CursorInfo/cursor_generated_interface.swift

```diff
@@ -105,13 +105,13 @@ public class CType {}
 @interface DType
 @end
 
-// RUN: %sourcekitd-test -req=interface-gen-open -module LibA -- -F %t/frameworks -target %target-triple == -req=cursor -pos=14:57 -print-raw-response | %FileCheck %s --check-prefix=CHECKD
```

Comment:
This is `D` as in the `D` module, no need to rename.

Reply:
Ah, sorry, thanks! I've reverted. (I think that is what you meant, right?)